VKClientRM/VKClientRM/Model/newsfeedGet/Newsfeed.swift

```diff
@@ -0,0 +1,43 @@
+// Newsfeed.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Новость пользователя
+final class Newsfeed: Decodable {
+    /// тип списка новости
+    let type: PostTypeEnum
+    /// идентификатор источника новости
+    let sourceID: Int
+    /// время публикации новости
+    let date: Int
+    /// фотографии
+    let photos: Photos?
+    /// текст записи
+    let text: String?
+    /// лайки новости
+    let likes: PurpleLikes?
+    /// просмотри новости
+    let views: Views?
```

Comment:
```suggestion
    /// тип списка новости
    let type: PostTypeEnum
    /// идентификатор источника новости
    let sourceID: Int
    /// время публикации новости
    let date: Int
    /// фотографии
    let photos: Photos?
    /// текст записи
    let text: String?
    /// лайки новости
    let likes: PurpleLikes?
    /// просмотри новости
    let views: Views?
```
Все с заглавной буквы(проперти)